src/commands/console/open.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-console:open', { provider: 'debug' })
+const { getCliEnv } = require('@adobe/aio-lib-env')
+const { CliUx } = require('@oclif/core')
+const { OPEN_URLS } = require('../../config')
+
+const ConsoleCommand = require('./index')
+class OpenCommand extends ConsoleCommand {
+  async run () {
+    // TODO: we could support --orgId and --projectId flags to open the console for a specific org/project
+    // currently we open the console for the local config org/project
+    // const { flags } = await this.parse(OpenCommand)
+
+    aioLogger.debug('Inquiring currently selected Org, Project and Workspace')
+    const cliEnv = getCliEnv()
+    const config = this.getConfig()
+    let url = OPEN_URLS[cliEnv]
+    if (config?.project?.id && config?.project?.org_id) {
```

Comment:
What happens when no Project or Org has been selected ? I guess it opens the default console URL `https://developer.adobe.com/console/projects`. 
Should we log a message in CLI that no org/project has been selected and then open the above URL? So that user can select one and not get confused why their specific project in not opening. WDYT?

Reply:
I would be in favor of prompting the user to select a project as we do when they run `aio app use`

Reply:
```
  test('should open a browser with default view if no project/workspace selected', async () => {
    config.get.mockReturnValue(null)
    await expect(command.run()).resolves.not.toThrowError()
    expect(cli.open).toHaveBeenLastCalledWith('https://developer.adobe.com/console/projects')
  })
```